peep.c

```diff
@@ -1226,6 +1226,9 @@ S_finalize_op(pTHX_ OP* o)
     PERL_ARGS_ASSERT_FINALIZE_OP;
 
     do {
+if (o->op_type == OP_FREED) {
+PerlIO_stdoutf("BUGGER! 0x%p is ->op_type == OP_FREED\n", o);
+}
```

Comment:
This should probably be removed :)

Reply:
Gah. Can't believe I missed these debugging statements. Thanks for your eagle eyes!